<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="displaySqlQueryTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>821</width>
      <height>461</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <column>
     <property name="text">
      <string>Employee_ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>First_name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>last_name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>address</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Phone_No</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Job_title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Salary</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="addPushButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>52</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletePushButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>136</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editPushButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>220</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findPushButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>304</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshPushButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>390</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1005</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
